css-media-queries/breakpoints.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Media Queries - Breakpoints</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+    }
+
+    .card-wrapper {
+      padding: 0.5rem;
+      flex-basis: 100%;
+    }
+
+    /* your code here 👇 */
+    @media (width > 575px) {
+      .card-wrapper {
+        flex-basis: 50%;
+      }
+    }
+
+    @media (width > 767px) {
```

Comment:
Just be mindful of the Range Syntax and that while, mostly supported it is not as supported as queries like `min-width: 575px` 

https://caniuse.com/?search=%40media

Namely, IE 11 does not support range queries at all.